source/clear-linux/get-started/bare-metal-install/bare-metal-install.rst

```diff
@@ -1,286 +1,336 @@
+The next menu of the |CL| installer setup is ``Choose installation Type`` and
+is shown in figure 7. This menu will allow you to install |CL|
+**automatically** or **manually**. You can also terminate the |CL| Installer
+to stop the |CL| installer setup process by selecting :guilabel:`< Exit >` and
+shutting down the system.
+
+.. figure:: figures/cai4of6.png
+   :scale: 50 %
+   :alt: Choose installation type
+
+   Figure 7: :guilabel:`Choose installation Type`
+
+Manual installation
+===================
+
+If you select :guilabel:`< Automatic >` as the installation type, the |CL|
+Installer will add the minimum amount of functionality required for a fully
+functional |CL| system. You will not be able to modify the disk layout, add
+a user or any other tasks that the manual installation process will allow.
+
+With the :guilabel:`< Manual(Advanced) >` option you can do the following
+additional tasks during |CL| Installer setup:
+
+* modify the disk layout using the cgdisk utility.
+* add additional command-line parameters to the kernel.
+* create a hostname for your system.
+* create an administrative user.
+* add additional software bundles to enhance the functionality of your initial
+  |CL| installation.
+* optionally, set up a static IP address for your system.
+
+To implement any of the additional items listed above for the
+manual option, select the :guilabel:`< Manual(Advanced) >` menu item and
+follow the steps outlined in the :ref:`bare-metal-manual-install`. This guide
+is a continuation from this point.
+
+.. toctree::
+   :maxdepth: 1
+
+   bare-metal-manual-install
+
+Automatic installation
+**********************
+
+If you only want to install the minimum components for your |CL|
+implementation the recommended action is to select the
+:guilabel:`< Automatic >` menu item shown in figure 7.
+
+#. Move your cursor to this field and press the :kbd:`Enter` key.
+
+#. This will bring you to the ``Choose target device for installation``
+   screen as shown in figure 8. Move your cursor to the desired target and
+   press the :kbd:`Enter` key.
+
+   .. figure:: figures/cai5of6.png
+      :scale: 50 %
+      :alt: Choose target device for installation
+
+      Figure 8: :guilabel:`Choose target device for installation`
+
+   In this example we select the single primary partition from our hard drive.
+
+#. With all the |CL| installer setup information gathered for the automatic
+   installation option, the |CL| Installer will prompt you to begin the actual
+   installation as shown in figure 9.
+
+   .. figure:: figures/cai6of6.png
+      :scale: 50 %
+      :alt: Begin installation
+
+      Figure 9: :guilabel:`Begin installation`
+
+   If you are satisfied with the information you have entered, select the
+   :guilabel:`< Yes >` field and press :kbd:`Enter` to begin installing |CL|.
+
+|CL| Installation begins...each step will show it's status as it progresses
+through the automated installation process.
+
+Complete installation process

```

Comment:
Why is this a separate section? These steps are still part of the automatic installer and should be included on that section.